docs/modules/reference/pages/provisioning/handlers/introduction.adoc

```diff
@@ -98,4 +98,10 @@ NOTE: To create an advanced cron schedule, click the checkbox and add the approp
 .. No scanning
 
 . The optional *Advanced Options* section lets you specify key/value pairs.
-. Click *Save & Close*.
\ No newline at end of file
+. Click *Save & Close*.
+
+Note the following field validation information:
+
+* Zone and Requisition fields cannot start with a space
```

Comment:
not sure if this worth to mention: fields cannot end with a space, but can have space in between